CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 ## Unreleased
 * Fixed an issue where the **coverage-analyze** command was not parsing the logs correctly.
 * Fixed an issue where **validate** falsly failed with error `DS108` on descriptions ending with a newline.
+* Added formatting for script yml files when period is missing in the end of comment field, in the **format** command.
+* Fixed an issue where **format**  add a newline with a period when the description field missing a period.
 * The content graph will now include the **python_version** field that each script/integration uses.
 * Updated the **update-release-notes** command message structure when is run with **--force** flag.
 * Added support for lazy loading the of widgets in XSIAM dashboards.
 
 ## 1.20.1
-* Added formatting for yml files when period is missing in the end of description field, in the **format** command.
+* Added formatting for integration yml files when period is missing in the end of description field, in the **format** command.
```

Comment:
Are we ok with changing it after release?

Reply:
you are right, i will del this